templates/includes/header.html.twig

```diff
@@ -0,0 +1,44 @@
+<header id="header" class="header" role="banner" aria-label="{{ 'Site header'|t}}">
+  {% block head %}
+    {% if page.secondary_menu or page.top_header or page.top_header_form %}
+      <nav{{ navbar_top_attributes.addClass('ezcontent-header') }}>
+      {% if container_navbar %}
+      <div class="container tt">
+      {% endif %}
+          {{ page.secondary_menu }}
+          {{ page.top_header }}
+          {% if page.top_header_form %}
+            <div class="form-inline navbar-form float-right">
+              {{ page.top_header_form }}
+            </div>
+          {% endif %}
+      {% if container_navbar %}
+      </div>
+      {% endif %}
+      </nav>
+    {% endif %}
+    <nav{{ navbar_attributes.addClass('ezcontent-header') }}>
+      {% if container_navbar %}
+      <div class="container ttt">
```

Comment:
Same here, class name `ttt`. Is this correct ?

Reply:
Removed this class also